meta-mender-core/recipes-mender/mender-client/mender_3.99.0+really4.0.0-alpha1.bb

```diff
@@ -0,0 +1,37 @@
+require mender-client-cpp.inc
+
+################################################################################
+#-------------------------------------------------------------------------------
+# THINGS TO CONSIDER FOR EACH RELEASE:
+# - SRC_URI (particularly "branch")
+# - SRCREV
+# - DEFAULT_PREFERENCE
+#-------------------------------------------------------------------------------
+
+SRC_URI = "gitsm://github.com/mendersoftware/mender;protocol=https;branch=feature-c++-client"
+
+# Tag: 3.99.0+really4.0.0-alpha1
+SRCREV = "40926f656b8c5df605ee84b32dacc548c3935835"
+
+# Enable this in Betas, and in branches that cannot carry this major version as
+# default.
+# Downprioritize this recipe in version selections.
+DEFAULT_PREFERENCE = "-1"
```

Comment:
Hmm, don't we want it as the default in this branch?

Reply:
I guess there is no good reason on this special branch. Adjusted.